OADB/EMCAL/OADB_information.txt

```diff
@@ -0,0 +1,10 @@
+The EMCal OADB files have been moved to the EOS directory /eos/experiment/alice/analysis-data/OADB/EMCAL which is accessible via lxplus.
+Analysis tasks can access these files via, for example: AliDataFile::GetFileNameOADB("EMCAL/EMCALTimeL1PhaseCalib.root") which has been implemented into the Tender and the Correction Framework as well as most analysis tasks in AliPhysics.
+If you want to have the OADB files locally, you can download them from lxplus via:
+export OADB_PATH=/path/to/my/local/oadb
+rsync -av --delete cern_user@lxplus.cern.ch:/eos/experiment/alice/analysis-data/ $OADB_PATH/
+In order for local tests to work properly, please add the OADB_PATH to your bashrc or similar.
+
+In addition, a short history of changes to the files in EOS will be listed here:
+
+20180213: Moved all files to EOS.
```

Comment:
Why not part of the README file of the correction framework so it gets displayed as well in alidoc?

Reply:
That is also a possibility which we can discuss in the calibration meeting. The current solution is just temporary.

Reply:
Doxygen checks for READMExyz.txt. Renaming the file accordingly, and moving it to PWG/EMCAL where you find the other files is sufficient that doxygen finds it (not too complicated operation). And having it in doxygen will be for the benefit of many users.

Reply:
As I said, this can be done later on. For this PR, it might be the best idea to have at least some file remain in the OADB/EMCAL folder so that people (who do not follow AliPhysics changes) know where the files were moved.